sklearn/calibration.py

```diff
@@ -511,7 +511,8 @@ def predict(self, T):
 
 
 def calibration_curve(y_true, y_prob, normalize=False, n_bins=5):
-    """Compute true and predicted probabilities for a calibration curve.
+    """Compute true and predicted probabilities for a calibration curve also
```

Comment:
The first line in the docstring should be a single line (keep forgetting the pep number for this).

Please add the sentence as a separate remark.